gst/nnstreamer/elements/gsttensor_merge.c

```diff
@@ -380,6 +380,8 @@ static gboolean
 gst_tensor_merge_get_merged_config (GstTensorMerge * tensor_merge,
     const GstTensorsConfig * in_config, GstTensorsConfig * out_config)
 {
+  GstTensorsInfo *in_info;
+  GstTensorInfo *_info;
```

Comment:
You've done a great job with the code. 👍 
Is there any other variable that can differentiate between tensorsInfo and tensorInfo?
It might be a bit difficult to think about.
```tensors_info``` and ``` tensor_info``` ?  The variable name is long. Is this okay? How do you feel about this?

Reply:
_info for temporal indexed ptr of input or output info